### ISSUE TITLE
Support multilabel classification

### ISSUE THREAD

**Issue by sandypreiss** [pmbaumgartner/fasttext-lite#1]
This would be useful for me personally, and I think a nice addition to this awesome package.

A natural approach would be to follow the example of sklearn's [MultiOutputClassifier](https://github.com/scikit-learn/scikit-learn/blob/364c77e04/sklearn/multioutput.py#L352).
- `FastTextMultiOutputClassifer` would inherit from `FastTextClassifier`
- Changes to `FastTextMultiOutputClassifer.fit`
    - Take `Y` as an array-like of shape (n_samples, n_classes)
    - Take label names as a list-like of length n_classes
    - Write the temp file using the multilabel format expected by FastText, e.g. `__label__sauce __label__cheese How much does potato starch affect a cheese sauce recipe?`
- `loss` would be fixed at `ova`
- I think `predict_proba` wouldn't need any changes, since FastText returns a sorted list of label probabilities either way.

If this sounds reasonable, I'm happy to write it.

**Comment by pmbaumgartner** [comment]
<img src="https://media.giphy.com/media/wi8Ez1mwRcKGI/giphy.gif">

**Comment by pmbaumgartner** [comment]
> * I think `predict_proba` wouldn't need any changes, since FastText returns a sorted list of label probabilities either way.

Just be careful on this one - it returns them in descending order of value, so the logic there means that we have to re-sort them in the order the labels are provided. I don't think anything would have to change, but it would be good to test for as well.
